spec/controllers/embed_controller_spec.rb

```diff
@@ -20,11 +20,15 @@
 
     before do
       Fabricate(:embeddable_host)
+      SiteSetting.stubs(:embed_link_to_top).returns(true)
```

Comment:
You don't need to stub site settings in tests. It's better to do `SiteSetting.embed_link_to_top = true`